### PR TITLE
add option to enter Incantation Forrest

### PR DESCRIPTION
**Pull request by CarlosEAM**
Please review this, @casey-collab

### CODE REVIEW

File: english/alchemy/alchemy.md [hunk 1]
```diff
@@ -38,5 +38,7 @@ You slip the ring over your finger and ...
 
 [Transform!](../transform/transform.md)
 
+[You teleport to the Incantation Forrest](../incantation/incantation-forrest.md)
```
Comment (casey-collab) [inline english/alchemy/alchemy.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/incantation-forrest/incantation-forrest.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Everything is black and you begin to hear the incantation softly in your ears:
```
Comment (casey-collab) [inline english/incantation-forrest/incantation-forrest.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by CarlosEAM** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :purple_heart:

**Pull request merged**